server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -190,6 +190,7 @@ enum CompactionState {
   private CompactableImpl compactable;
 
   private volatile CompactionState minorCompactionState = null;
+  private volatile boolean lastMinCSuccessful = true;
```

Comment:
Could remove this if the code that checks it in close is removed.

Reply:
Removed in 31260e0